src/main/java/mezz/jei/render/IngredientListBatchRenderer.java

```diff
@@ -156,6 +157,8 @@ public void render(Minecraft minecraft, MatrixStack matrixStack) {
 		TextureManager textureManager = minecraft.getTextureManager();
 		itemRenderer.zLevel += 50.0F;
 
+		IRenderTypeBuffer.Impl buffer = Minecraft.getInstance().getRenderTypeBuffers().getBufferSource();
```

Comment:
Given minecraft is one of the args of this method wouldn't it make more sense to use that instead of `Minecraft.getInstance()`